recipes/bw_graph_tools/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge/label/bw2calc_dev,conda-forge
```

Comment:
Typically we don't allow "dev dependencies" (except in very rare circumstances).

Usually we would allow an established package to upload a dev version to a dev channel for testing purposes.

Are you able to build without this for the first version?